File: Java/AdvProg/Lab3/src/questions/Question1.java
package questions;

public class Question1 {

	public static void main(String[] args) {
		String [] devices = {"Keyboard", "Monitor", "Printer", "Server"};
		
		for(int i = 0; i < devices.length; i++) {
			System.out.println("Element at index " + i + ":" + devices[i] + " | " + devices[i].length() +  "chars");
		}
	}

}

File: repos/Lab3CSharp/Lab3CSharp/Question1.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Lab3CSharp
{
    class Question1
    {
        public string[] array = { "Keyboard", "Monitor", "Printer", "Server"};

        
        public void qOne()
        {
            for (int i = 0; i < array.Length; i++)
            {
                Console.WriteLine("Element and index " + i + ": " + array[i] + " | " + array[i].Length);
            }
        }
        
    }
}
